drf_problems/exceptions.py

```diff
@@ -31,15 +31,27 @@ def exception_handler(exc, context):
     problem_title = getattr(exc, 'title', exc.default_detail)
     problem_status = response.status_code
     problem_code = getattr(exc, 'code', exc.default_code)
-    problem_type = reverse('drf_problems:error-documentation',
-                           kwargs={'code': problem_code}, request=request)
+    problem_type = reverse(
+        'drf_problems:error-documentation',
+        kwargs={'code': problem_code},
+        request=request,
+    )
+    problem_instance = getattr(exc, 'instance', None)
```

Comment:
If the `instance` attribute is an endpoint, then the full URL should be added in the error detail